catalog/MDCCatalog/MDCMenuViewController.swift

```diff
@@ -0,0 +1,59 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import MaterialComponents.MDCIcons
+
+class MDCMenuViewController: UITableViewController {
+
+  let tableData =
+    [("Settings", MDCIcons.imageFor_ic_settings()?.withRenderingMode(.alwaysTemplate)),
+     ("Themes", MDCIcons.imageFor_ic_color_lens()?.withRenderingMode(.alwaysTemplate)),
+     ("Help", MDCIcons.imageFor_ic_help_outline()?.withRenderingMode(.alwaysTemplate))]
+  let cellIdentifier = "MenuCell"
+  let iconColor = UIColor(red: 100/255.0, green: 100/255.0, blue: 100/255.0, alpha: 1)
```

Comment:
Can this be pulled from the app theme somehow?

Reply:
Done.